vagga.yaml

```diff
@@ -2,19 +2,41 @@ containers:
   pip-freeze:
     setup:
       - !Alpine v3.4
+      - !BuildDeps
+        - postgresql-dev
```

Comment:
`!BuildDeps` are removed after container is built. `!Install` packages are available in runtime.
But the question here is that you install `psycopg2` python package and it requires some package during build time which you don't provide in `django` container. This means that `libpq` is enough for build, so why not use the same package here?